stable/spinnaker/templates/configmap/spinnaker-config.yaml

```diff
@@ -6,7 +6,107 @@ metadata:
     app: {{ template "fullname" . }}
 data:
   clouddriver.yml: |
-{{.Files.Get "config/clouddriver.yml" | indent 4}}
+    server:
```

Comment:
What's the reasoning behind mixed mode Files.Get and having the file embedded in the configMap?